applications/serial_lte_modem/doc/nRF91_as_Zephyr_modem.rst

```diff
@@ -0,0 +1,103 @@
+.. _slm_as_zephyr_modem:
+
+nRF91 Series as a Zephyr-compatible modem
+#########################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+Starting with |NCS| v2.6.0, the Serial LTE Modem (SLM) application can be used to turn an nRF91 Series SiP into a standalone modem that can be used through Zephyr's cellular modem driver.
+This means that the controlling chip can run a Zephyr application that seamlessly uses Zephyr's IP stack instead of offloaded sockets via AT commands.
+
+This is made possible by SLM's support of CMUX and PPP and Zephyr's cellular modem driver.
```

Comment:
Can you rewrite this section.
From 2.6.0 the Serial LTE Modem supports PPP and CMUX (standard?).

This means that it can be used through Zephyr's cellular modem driver.

Or something like that.. There are other users than Zephyr, so we should first mention that we are supporting PPP+CMUX and only after that, mention that we are compatible with Zephyr.